bin/birdhouse

```diff
@@ -561,6 +561,18 @@ parse_backup_create_args() {
       parse_backup_create_args "$@"
       ;;
     '')
+      if
+        [ "${BIRDHOUSE_BACKUP_NO_RESTIC}" = 'true' ] && \
+        [ -z "${BIRDHOUSE_BACKUP_VOLUME}" ]; then
```

Comment:
This isn't set in this script (it can be set in `env.local` with a default in `birdhouse/default.env`) so at this point in the script this variable will always be unset.

I would recommend doing these checks in the `do_backup_create_restore` since the variable will be loaded by that point.

Reply:
I was invoking like this `BIRDHOUSE_BACKUP_VOLUME=/tmp/backup birdhouse backup [...]`. 

I wanted to purposely avoid the `default.env` in this case because that will cause the "potentially erroneous warning" case right after when using a named-volume by default. Do we want that anyway? I though it was preferable that the variable was supplied explicitly in this specific case.